shared/lib/constants.py

```diff
@@ -375,18 +375,27 @@
 # are considered as a match.
 SV_SCORE_DEFAULT_THRESHOLD = 0.5
 
-# The default Cosine score threshold beyond which Stat Vars
-# are considered a high confidence match.
-SV_SCORE_HIGH_CONFIDENCE_THRESHOLD = 0.7
+# The Cosine score bump (aka increase over default model-threshold)
+# to be applied for a high confidence match.
+# See compute_final_threshold()
+#
+# bump of 0.4 => a threshold of 0.7 if default is 0.5
+#             => a threshold of 0.82 if default is 0.7
```

Comment:
for the two example here, how is 0.4 used? Have more explanation?

Reply:
Added more comments, ptal?

Reply:
ah, i see.